Apps/Playground/Scripts/experience.js

```diff
@@ -225,6 +227,56 @@ CreateBoxAsync(scene).then(function () {
                     }, 5000);
                 }
 
+                const fm = xr.baseExperience.featuresManager;
+                console.log(fm);
+                const anchors = fm.enableFeature(BABYLON.WebXRFeatureName.ANCHOR_SYSTEM, 'latest');
+                const xrPlanes = fm.enableFeature(BABYLON.WebXRFeatureName.PLANE_DETECTION, 'latest');
+                const xrTest = fm.enableFeature(BABYLON.WebXRFeatureName.HIT_TEST, 'latest');
+                const xrBackgroundRemover = fm.enableFeature(BABYLON.WebXRFeatureName.BACKGROUND_REMOVER, 'latest');
+
+                var logOnce = true;
+                var hitTest;
+                xrTest.onHitTestResultObservable.add((result) => {
+                    //console.log("xrTest.onHitTestResultObservable ", result.length);
+                    if (result.length && logOnce) {
+                        console.log("got hit test");
+                        logOnce = false;
+                        //marker.isVisible = !isPlaced;
+                        // marker.isVisible = true;
+                        hitTest = result[0];
+                        /*marker.position = hitTest.position;
```

Comment:
Seems like there is some useful test code here, but also a bunch of stuff commented out. Are you going to clean this up and include it in the merge?

Reply:
no, the plan is to remove this from the PR once it's approved. The code might be a good start to continue working on this issue : https://github.com/BabylonJS/BabylonNative/issues/702